### PR TITLE
Refactor seeder, prepare for tests

### PR DESCRIPTION
**Pull request by Implausibility**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  // setupFiles: [],
+
+  // A list of paths to modules that run some code to configure or set up the testing framework before each test
+  // setupFilesAfterEnv: [],
+
+  // A list of paths to snapshot serializer modules Jest should use for snapshot testing
+  // snapshotSerializers: [],
+
+  // The test environment that will be used for testing
+  testEnvironment: "node",
```
Comment (eLeiLei) [inline jest.config.js:135]: When you get to testing the front-end, you'll need a browser-like test environment. Jest has a default for the front-end.

Comment (Implausibility) [reply jest.config.js:135]: Yeah, I wondered how to have that one and the node tests at the same time. I'll be looking into that today.

**Comment by Implausibility** [comment]
Thanks, @eLeiLei !

**Comment by jessebeard** [comment]
took me a minute to wrap my head around what your code was doing but I got it! lgtm! it might be nice to add a comment at the top of your functions that explains what each function should return

**Comment by Implausibility** [comment]
@jessebeard Good idea, I'll do that.

**Pull request merged**